src/styl/_base/_typography.styl

```diff
@@ -33,12 +33,14 @@ h6
     _fontload($fontstack-headings)
     font-weight bold
     line-height 1.2
+    line-height $lh
 
 h1
     font-size _rem(21px)
+    line-height 1.5 * $lh
 
 h2
-    font-size _rem(16px)
+    font-size _rem(17px)
```

Comment:
Is that a personal choice or a specific request ?

Reply:
Related to design support submited about new line-height